### ISSUE TITLE
Incorrect file name displayed in tooltip

### ISSUE THREAD

**Issue by thinklog** [atom/tabs#189]
The tab has a label that displays the file name. If you hover the cursor over that label a balloon displays the full path to the file. At times the balloon displays the wrong path, as in the following example.
1. Save "file 1" to "path 1".
2. Hover over the label.
3. The correct path of file 1 at path 1 is displayed.
4. Save As... the same "file 1" to "path 2", using the same file name. Only the path is different.
5. Hover over file name label. The displayed path is file 1 at path 1. 
6. Modify contents of file and save it.
7. Hover over the label. The displayed path is still file 1 at path 1.
8. The file that was modified, however, is file 1 at path 2!

A variation. If you change the file name at step 4, where you "Save As..." to path 2, then the label in the tab will display the new file name, but if you hover over the label, then the balloon will display the old file name with the old path.

In short, once set, information displayed in the file name and path information balloon doesn't change, even under circumstances where the label correctly displays the new file name. Moreover, regardless what path is displayed in the balloon, modifications will always be saved to the latest file name and path combination for that tab.

Hope this helps.

**Comment by mnquintana** [comment]
More steps to reproduce:

1. Open a file.
1. Move it inside another directory with `Tree View: Rename`
1. Hover over tab - the old path will be displayed.

**Comment by bradleyflood** [comment]
Same issue when using `Save as`, the old filename is displayed in the balloon.

**Comment by hgGeorg** [comment]
Not only is the tooltip affected. The `data-name` attribute remains unchanged too.
